migrations/20240501124220_not_null_token_values.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
```

Comment:
Oh one more thing—we try to name this sequentially. I think goose has a command that helps you do that.